.ci/travis/travis.sh

```diff
@@ -293,12 +293,14 @@ no-error-xwiki)
   checkout_from https://github.com/xwiki/xwiki-rendering.git
   cd .ci-temp/xwiki-rendering
   # Validate xwiki-rendering
+  git reset --hard HEAD~3
   mvn -e --no-transfer-progress checkstyle:check@default -Dcheckstyle.version=${CS_POM_VERSION}
   cd ..
   removeFolderWithProtectedFiles xwiki-rendering
   cd ..
   checkout_from https://github.com/xwiki/xwiki-platform.git
   cd .ci-temp/xwiki-platform
+  git reset --hard HEAD~3
```

Comment:
Temporary change until Xwiki is fixed.

Reply:
Why not use exact commit?

Reply:
> Why not use exact commit?

Nice idea, separated into a different commit.

Reply:
I mean use the exact commit for travis build, i.e. `b038fd863a9d23695c6ed187dd3c0c6ace3fca60`

Reply:
Ok, I will do that in the next commit. I didn't realize it will take so much time for them to fix. Thanks for the heads-up!